src/rules/effects/use-effects-lifecycle-interface.ts

```diff
@@ -59,10 +59,8 @@ export default ESLintUtils.RuleCreator(docsUrl)<Options, MessageIds>({
 
         if (interfaces.includes(interfaceName)) return
 
-        const {
-          implementsNodeReplace,
-          implementsTextReplace,
-        } = getImplementsSchemaFixer(classDeclaration, interfaceName)
+        const { implementsNodeReplace, implementsTextReplace } =
+          getImplementsSchemaFixer(classDeclaration, interfaceName)
```

Comment:
I was seeing the following report by `prettier` (here and in /guards):

<img width="671" alt="Prettier report" src="https://user-images.githubusercontent.com/11965907/131406571-e856b656-6813-4919-86de-9b89e83f6438.png">